### ISSUE TITLE
Accessibility checklist updates

### ISSUE THREAD

**Issue by Blackbaud-DanMiller** [blackbaud/skyux2-docs#921]

**Comment by Blackbaud-DanMiller** [comment]
https://github.com/blackbaud/skyux-theme/issues/134

**Comment by Blackbaud-DanMiller** [comment]
https://github.com/blackbaud/skyux-theme/issues/135

**Comment by blackbaud-BenWong** [comment]
We have an accessibility checklist here: https://developer.blackbaud.com/skyux/learn/accessibility/test
